### PR TITLE
docs: migrate toh-pt5 to standalone

### PR DESCRIPTION
**Pull request by bencodezen**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by bencodezen** [comment]
@MarkTechson @twerske @jessicajaniuk I'm trying out the ng schematics automated migration for this PR. The tests say that everything passes, but the syntax looks a bit different from the way I've been doing it (particularly in `main.ts`).

Any guidance on whether this should be the preferred method going forward would be a great help!

File: aio/content/examples/toh-pt5/src/main.ts [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 
-platformBrowserDynamic().bootstrapModule(AppModule)
+import { importProvidersFrom } from '@angular/core';
+import { AppComponent } from './app/app.component';
+import { AppRoutingModule } from './app/app-routing.module';
+import { FormsModule } from '@angular/forms';
+import { BrowserModule, bootstrapApplication } from '@angular/platform-browser';
+
+bootstrapApplication(AppComponent, {
+    providers: [importProvidersFrom(BrowserModule, FormsModule, AppRoutingModule)]
```
Comment (jessicajaniuk) [inline aio/content/examples/toh-pt5/src/main.ts:11]: You could switch this over to use `provideRouter` rather than `app-routing.module`.

**Review by jessicajaniuk — APPROVED** [review]
This is fine. The only suggestion is the `provideRouter` option.

File: aio/content/examples/toh-pt5/src/main.ts (hunk 1 above)
Comment (AndrewKushnir) [reply aio/content/examples/toh-pt5/src/main.ts:11]: +1, you could also drop `BrowserModule` and `FormsModule` references - there are no eager providers there that should be added.

Comment (MarkTechson) [reply aio/content/examples/toh-pt5/src/main.ts:11]: +1

@bencodezen let me know if any of this gives you any issues.

**Review by MarkTechson — CHANGES_REQUESTED** [review]

**Pull request closed without merging**